src/sync/bounded.zig

```diff
@@ -100,36 +102,34 @@ pub fn Bounded(comptime T: type) type {
 
             while (true) {
                 // Check if the channel is disconnected.
-                if (tail & self.disconnect_bit != 0) {
+                if (tail & disconnect_bit != 0) {
                     return error.disconnected;
                 }
 
                 // Deconstruct the tail.
-                const index = tail & (self.disconnect_bit - 1);
-                const lap = tail & ~(self.one_lap_bit - 1);
```

Comment:
Previously the lap was only used to determine how to increment the tail. This is not needed in the new implementation because the tail can always be incremented with +1.